Improved functions and R shiny code added

andreeaz19 opened the pull request:
Modified functions and added a new file with R shiny code so that the function can run in an R shiny app for users.

andreeaz19 committed:
Improved functions and R shiny code added

Modified functions and added a new file with R shiny code so that the function can run in an R shiny app for users.

andreeaz19 committed:
Correctly Added R shiny package

I realized I had not added the R shiny code to the folder, so this has now been added.

yassieq reviewed (commented):
Your app looks interesting! Nice that you havemultiple exportable functions and the code looks good. Don't forget to put bslib with the imports in the description file. Additionally, I was wondering if you are going to take out the functions from the app file, since they're already in the functions file qnd can be sourced from there. Also, there are now two R files with the same code. Another small thing is that there seems to be some overlap in your functions, for example:

country <- input$country

Is called multiple times. I was wondering if you could maybe shorten your code by calling this once in the beginning of your server function

andreeaz19 committed:
Added Final report draft

Added final report draft

andreeaz19 committed:
Improved Functions and Shiny App

Changed the R package functions so that the R shiny app works properly, and changed layout of shiny app.

andreeaz19 committed:
Updated function to open R shiny and updated README.

A new function has been added, open_app, which allows the user to open the R shiny application. A few changes have been made to improve the GUI, and an elaborated README file has been added with information on how to use the package.
